scripts/run_dev_simplified.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
I'd like to avoid forking the run_dev.sh script if possible. Could you help me understand what was problematic in the original script that made a simplified version appealing? Too much magic in resolving values or something else perhaps?

Reply:
Sorry about that @hemalshahNV that script should not have been committed.